precise/parrot_neuron_ps.h

```diff
@@ -34,38 +34,32 @@ Name: parrot_neuron_ps - Neuron that repeats incoming spikes handling
 precise spike times.
 
 Description:
-  The parrot neuron simply emits one spike for every incoming spike.
-  One possible application for this is to create different channels
-  for the output of generator devices such as the poisson_generator
-  or the mip_generator.
+The parrot neuron simply emits one spike for every incoming spike.
+An important application is to provide identical poisson spike
+trains to a group of neurons. The poisson_generator sends a different
+spike train to each of its target neurons. By connecting one
+poisson_generator to a parrot_neuron and then that parrot_neuron to
+a group of neurons, all target neurons will receive the same poisson
+spike train.
 
 Remarks:
-  Network-wise the parrot neuron behaves like other neuron models
-  regarding connections and communication. While the number of
-  outgoing spikes equals that of incoming ones, the weigth of the
-  outgoing spikes solely depends on the weigth of outgoing connections.
-
-  A Poisson generator that would send multiple spikes during a single
-  time step due to a high rate will send single spikes with
-  multiple synaptic strength instead, for effiacy reasons.
-  This can be realized because of the way devices are implemented
-  in the threaded environment. A parrot neuron on the other
-  hand always emits single spikes. Hence, in a situation where for
-  example a poisson generator with a high firing rate is connected
-  to a parrot neuron, the communication cost associated with outgoing
-  spikes is much bigger for the latter.
-
-  Please note that this node is capable of sending precise spike times
-  to target nodes (on-grid spike time plus offset). If this node is
-  connected to a spike_detector, the property "precise_times" of the
-  spike_detector has to be set to true in order to record the offsets
-  in addition to the on-grid spike times.
-
-Parameters:
-  No parameters to be set in the status dictionary.
-
-References:
-  No references
+
+- Weights on connection to the parrot_neuron are ignored.
+- Weights on connections from the parrot_neuron are handled as usual.
+- Delays are honored on incoming and outgoing connections.
+
+Only spikes arriving on connections to port 0 will be repeated.
+Connections onto port 1 will be accepted, but spikes incoming
+through port 1 will be ignored. This allows setting exact pre-
+and post-synaptic spike times for STDP protocols by connecting
+two parrot neurons spiking at desired times by, e.g., a
+stdp_synapse onto port 1 on the post-synaptic parrot neuron.
```

Comment:
I believe, this is not true, yet. You changed `parrot_neuron_ps::handles_test_event(...)` to accept connections to port 1, but `parrot_neuron_ps::handle(...)` is unchanged, adding all incoming events, not differentiating between ports.